packages/server/graphql/queries/notifications.ts

```diff
@@ -31,7 +31,7 @@ export default {
       .limit(first + 1)
       .run()
 
-    const nodes = nodesPlus1.slice(0, -1)
+    const nodes = nodesPlus1.slice(0, first)
```

Comment:
This is a bug - previously, we were looking up the first 6 notifications, then dropping the last element to get down to 5, then using the original to figure out some pagination.  But if a user has fewer than 5 notifications, we'll drop the last one.

Reply:
+2 i'm a moron